cms/envs/common.py

```diff
@@ -142,6 +142,11 @@
 
     # Enable course reruns, which will always use the split modulestore
     'ALLOW_COURSE_RERUNS': True,
+
+    # Social Media Sharing on Student Dashboard
+    'DASHBOARD_SHARE_SETTINGS': {
```

Comment:
NIT: I've seen elsewhere that we add comments for feature flags that should be set the same way in both LMS / CMS. That might be a helpful doc string addition to this.